libraries/effect/src/main/assets/shaders/fragment_shader_copy_es2_simple.glsl

```diff
@@ -0,0 +1,24 @@
+#version 100
+// Copyright 2024 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// ES 2 fragment shader that samples from a (non-external) texture with
+// uTexSampler and copies this to the output.
+
+#extension GL_OES_EGL_image_external : require
+precision mediump float;
+uniform samplerExternalOES uTexSampler;
```

Comment:
An external sampler must be used as compared to a `sampler2D`, although the actual usage is the same